h5p_proplist_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
```

Comment:
`s/2017/2019/`

Reply:
All files are with same license title. Shall we change it all with another PR?

Reply:
New files get their copyright year from the year they are written.
We don't update old files' copyright.

Reply:
OK. Fixed